bigtop-packages/src/common/zookeeper/install_zookeeper.sh

```diff
@@ -187,7 +187,7 @@ for pairs in zkServer.sh/zookeeper-server zkServer-initialize.sh/zookeeper-serve
 # Autodetect JAVA_HOME if not defined
 . /usr/lib/bigtop-utils/bigtop-detect-javahome
 
-export ZOOPIDFILE=\${ZOOPIDFILE:-/var/run/zookeeper/zookeeper_server.pid}
+export ZOOPIDFILE=\${ZOOPIDFILE:-/run/zookeeper/zookeeper_server.pid}
```

Comment:
This causes mismatch between /usr/bin/zookeeper-server and /etc/init.d/zookeeper-server on Debian and Ubuntu since generated scripts from [the template](https://github.com/apache/bigtop/blob/c111a3069aa54273b68309092595cb7d2fc052f4/bigtop-packages/src/templates/init.d.tmpl) is used. We can address the issue on follow-up JIRAs for systemd migration on DEB.

```
Error: /Stage[main]/Hadoop_zookeeper::Server/Service[zookeeper-server]/ensure: change from 'stopped' to 'running' failed: Systemd start for zookeeper-server failed!
journalctl log for zookeeper-server:
Oct 16 05:43:23 0560bc522478 systemd[1]: Starting ZooKeeper is a centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services....
Oct 16 05:43:23 0560bc522478 zookeeper-server[5040]: ZooKeeper JMX enabled by default
Oct 16 05:43:23 0560bc522478 zookeeper-server[5040]: Using config: /etc/zookeeper/conf/zoo.cfg
Oct 16 05:43:23 0560bc522478 zookeeper-server[5056]: mkdir: cannot create directory ???/run/zookeeper???: Permission denied
Oct 16 05:43:23 0560bc522478 zookeeper-server[5040]: Starting zookeeper ...
Oct 16 05:43:23 0560bc522478 zookeeper-server[5058]: /usr/lib/zookeeper/bin/zkServer.sh: line 175: /run/zookeeper/zookeeper_server.pid: No such file or directory
Oct 16 05:43:23 0560bc522478 zookeeper-server[5040]: FAILED TO WRITE PID
Oct 16 05:43:23 0560bc522478 systemd[1]: zookeeper-server.service: Control process exited, code=exited, status=1/FAILURE
Oct 16 05:43:23 0560bc522478 systemd[1]: zookeeper-server.service: Failed with result 'exit-code'.
Oct 16 05:43:23 0560bc522478 systemd[1]: zookeeper-server.service: Unit process 5057 (nohup) remains running after unit stopped.
Oct 16 05:43:23 0560bc522478 systemd[1]: Failed to start ZooKeeper is a centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services..

```